src/esn.calendar.libs/app/components/event/form/event-form.service.js

```diff
@@ -43,7 +43,7 @@ function calEventFormService($rootScope, $modal, calendarService, calUIAuthoriza
 
     modalIsOpen = true;
     $modal({
-      template: require("../modals/event-form-modal.pug"),
+      template: require("./modals/event-form-modal.pug"),
```

Comment:
We should we single quote

Reply:
Since we are using `"` in many places without lint it. I would keep it like this.